package.json

```diff
@@ -10,13 +10,14 @@
     "url": "sindresorhus.com"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
   },
   "scripts": {
-    "test": "xo && ava"
+    "test": "xo --ignore index.d.ts && ava"
```

Comment:
Do it like the example is shared: https://github.com/sindresorhus/delay/blob/c17e29b13c20e6be35d64d75f45ce174bb793158/package.json#L48-L52

Reply:
👍 Sounds good!